js/src/hooks/useCountryKeyNameMap.js

```diff
@@ -4,6 +4,18 @@
 import { SETTINGS_STORE_NAME } from '@woocommerce/data';
 import { useSelect } from '@wordpress/data';
 
+/*
+ * Examples:
+ * - Cura&ccedil;ao                             -> Curaçao
+ * - Saint Barth&eacute;lemy                    -> Saint Barthélemy
+ * - S&atilde;o Tom&eacute; and Pr&iacute;ncipe -> São Tomé and Príncipe
+ */
+function replaceHtmlEntities( countryName ) {
+	const el = document.createElement( 'span' );
+	el.innerHTML = countryName;
+	return el.textContent;
+}
+
```

Comment:
:nail_care: WDYT of using https://www.npmjs.com/package/@wordpress/html-entitiesm? It's somewhat test-covered.

Reply:
Replaced by https://github.com/woocommerce/google-listings-and-ads/pull/917/commits/2a59a994a56802d1b828c05136cab28683757ced